gnome-base/gnome-panel/gnome-panel-3.46.0.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit gnome2 toolchain-funcs
+
+DESCRIPTION="The GNOME Flashback Panel"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-panel/"
+
+LICENSE="GPL-2+ FDL-1.1 LGPL-2.1+"
+SLOT="0"
+IUSE="eds elogind systemd"
+REQUIRED_USE="^^ ( elogind systemd )"
+KEYWORDS="~amd64 ~riscv"
+
+# eds-3.46.0[weather] and libsoup:3 because of build failure https://gitlab.gnome.org/GNOME/gnome-panel/-/issues/41
+# eds build with USE="weather" ensures eds is built against libsoup:3.
+# eds built with USE="-weather" could be build with libsoup:2.4 because it does not need libgweather-4.2.0
+# This would result in a build failure in gnome-panel
```

Comment:
I'm confused by this analysis, because I don't see that eds-3.46.0 has any dependence on libsoup:2.4. If I understand correctly, it has no dependence on libsoup at all with `USE=-weather`.

Reply:
Yes I commented on this further up https://github.com/gentoo/gentoo/pull/27596#discussion_r987633522
I don't really know how to handle this.